content/AppleAuth gem integration with Ruby on Rails - Antonieta Alvarez.md

```diff
@@ -0,0 +1,56 @@
+# AppleAuth gem, Rails Apple Sing In integration
```

Comment:
To help the discovery of the article from Google, we should brainstorm some other titles and the name that Apple gives is Sign in with Apple https://developer.apple.com/sign-in-with-apple/:
- Sign in with Apple in Ruby on Rails using the AppleAuth gem
- How to integrate Sign in with Apple in Ruby on Rails using the AppleAuth gem?
- Easily integrate Sign in with Apple in Ruby on Rails with AppleAuth gem

Reply:
- AppleAuth, the rails gem to easily configure Apple Sign In

Reply:
* AppleAuth, an easy ruby way to integrate with Apple Sign In.